footprints/batch/forms.py

```diff
@@ -35,10 +36,14 @@ class CreateBatchJobForm(forms.Form):
     csvfile = forms.FileField(required=True)
 
     def csvfile_reader(self):
-        csv_file = self.cleaned_data['csvfile']
-        csv_file.seek(0)
-        as_string = io.StringIO(csv_file.read().decode('utf-8'))
-        return csv.reader(as_string)
+        if sys.version > '3':
```

Comment:
I merged this, but this switch isn't quite right. On python 2, sys.version > '3' also evaluates to True:

```
>>> sys.version > 3
True
>>> sys.version
'2.7.16+ (default, Sep  4 2019, 08:19:57) \n[GCC 9.2.1 20190827]'
```